app/views/events/show.html.erb

```diff
@@ -1,31 +1,32 @@
-<div class="card mt-2">
-  <div class="card-header bg-primary">
-    <h1 class="text-center text-white fw-bolder">
-      <%= @event.name %>
-    </h1>
-    <p class="text-center text-white">
-      Időpontja: <%= l @event.start %> - <%= l @event.end %>
-    </p>
+<div class="my-3">
+  <h2>
+    <%= @event.name %>
+  </h2>
+  <p>
+    Időpontja: <%= l @event.start %> - <%= l @event.end %>
+  </p>
+</div>
+
+<div class="py-5">
+  <div class="mb-4 mt-2">
+    <%= @event.description %>
   </div>
-  <div class="card-body">
-    <div class="mb-4 mt-2">
-      <%= @event.description %>
+  <% if @event.event_type %>
+    <div class="d-flex justify-content-end mb-2">
+      <%= link_to 'Jelentkezek!', register_to_event_path(@event), class: "btn btn-outline-info" %>
```

Comment:
This bright blue feels a bit weird, because we don't use the color anywhere else